feature/metis/manage-conversations/src/main/kotlin/de/tum/informatics/www1/artemis/native_app/feature/metis/manageconversations/ui/common/CommunicationColors.kt

```diff
@@ -0,0 +1,14 @@
+package de.tum.informatics.www1.artemis.native_app.feature.metis.manageconversations.ui.common
+
+import androidx.compose.foundation.isSystemInDarkTheme
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.graphics.Color
+
+object CommunicationColors {
+    private val ArtemisMainBlue = Color(0xFF3E8ACC)
```

Comment:
I also discussed this with Eylül on Tuesday, I will try to see whether we can get closer to the "original" artemis colors by updating the Material 3 theme with a different seed color and / or different settings. As soon as I know more (hopefully today, but latest tomorrow), I will reach out to @eylulnc and discuss how we can tackle this :)

Reply:
So I checked with the official [Material theme builder](https://material-foundation.github.io/material-theme-builder/) and using the ArtemisMainBlue color as a seed color (for the current theme used in the app, we used a slightly different seed color). Depending on whether you use the "Color match" option, the results looks like this:
## Without Color match
![image](https://github.com/user-attachments/assets/232663bb-5b76-4cbf-9e54-40310393d12d)

## With Color match
![image](https://github.com/user-attachments/assets/940205d9-b8ed-48a8-9777-346750d09016)


With the color match option enabled, we get quite close to the seed color by using the PrimaryContainer color. 

In my opinion, this might be a good compromise between bringing the mobile app appearance closer together, while still complying to the Android Material3 design guidelines and keeping an Android look and feel. What do you think about that @eylulnc, @julian-wls ?

Reply:
I think it look great, colors are more vibrant and it feels closer to the iOS as well. Also as you said we are still match to the Material3.

Reply:
I think this would be a great solution.

Reply:
Awesome, then lets use this solution and primaryContainer as the color for this PR, I will create a PR in the meantime to update the Theme accordingly :D